apps/bash-script/bash-script-buildpack/buildpack.toml

```diff
@@ -3,7 +3,7 @@ api = "0.2"
 
 # Buildpack ID and metadata
 [buildpack]
-id = "io.buildpacks.samples.bash-script"
+id = "buildpacks-io-samples/bash-script"
```

Comment:
could this just be `samples/bash-script`?

Reply:
Made these changes in this PR: https://github.com/buildpacks/samples/pull/71/files